tests/api2/test_036_ad_ldap.py

```diff
@@ -24,8 +24,16 @@
     pytestmark = pytest.mark.skipif(dev_test, reason='Skip for testing')
 
 @pytest.fixture(scope="module")
-def kerberos_config():
-    return {}
+def kerberos_config(request):
+    payload = {"v4_krb": True}
+    results = PUT("/nfs/", payload)
+    assert results.status_code == 200, results.text
+    try:
+        yield(request)
```

Comment:
This looks strange? just yield request?

Reply:
I don't really need the config info, but yielding lets me check the status of request.